docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
+Try out the following instructions and see if you're actually able to open up and use a copy made by BitShelter.
 
 #### Oh no. Something happened to my Collection. How to save the day ?
 
-Don't panic ! Follow [this guide](https://www.howtogeek.com/howto/11130/restore-previous-versions-of-files-in-every-edition-of-windows-7/) to restore your collection to an earlier version.
+Don't panic! Restoring your collection to an earlier version is easy with BitShelter! 
+
+1. Right click on your collection folder and click on **Restore previous versions**
+
+<img src="/content/images/backup-setup/bitshelter-restorepre.png" data-origin="content/images/backup-setup/bitshelter-restorepre.png" alt="">
 
-<img src="/content/images/backup-setup/bitshelter-restore-previous-version.png" data-origin="content/images/backup-setup/bitshelter-restore-previous-version.png" alt="">
+2. Click on the copy you'd like to go back to:
+
+<img src="/content/images/backup-setup/bitshelter-restoremenu.png" data-origin="content/images/backup-setup/gitsetup-bitshelter-restoremenu" alt="">
+
+3. You can then click restore, to overwrite current version of the folder, or click open to browse the files:
+
+<img src="/content/images/backup-setup/bitshelter-restoremenu2.png" data-origin="content/images/backup-setup/gitsetup-bitshelter-restoremenu2" alt="">
+
+!> If you have issues with the above instuctions, you can also refer to [this guide](https://www.howtogeek.com/howto/11130/restore-previous-versions-of-files-in-every-edition-of-windows-7/) <img src="/content/images/backup-setup/bitshelter-restore-previous-version.png" data-origin="content/images/backup-setup/bitshelter-restore-previous-version.png" alt="">
 
 ### Internet backups: Git & Github
 
 #### Git 101
 
 ?> **Git** is a tool that makes it easier to track changes to files. When you edit a file, git can help you determine exactly *what* changed, *who* changed it, and *why*. <br /><br />It is useful for coordinating work among multiple people on a project, and for tracking progress over time by saving “checkpoints”. You could use it while writing an essay, or to track changes to artwork and design files. [\[1\]](https://hackernoon.com/understanding-git-fcffd87c15a3)
 
-For our purpose, **git** will be our mean to:
+**For our purposes**, ***git*** will be our mean to:
 - Create versions of our collection (*\"checkpoints\"*).
 - Upload\* our work to a safe\*\*, remote place.
 - *(Bonus)* Synchronize our work between devices.
 
-\*: GitHub has a 1GB limit on free private repositories.
-\*\*: *Although GitHub can be considered fairly reliable, we highly recommend that you implement a solution to encrypt your collection. This is, after all, a way to your most inner thoughts.*
+\*: GitHub has a 1GB limit on free private repositories. [GitLab](https://about.gitlab.com) offers 10gb free storage by default and [BitBucket](http://bitbucket.org) offers a 5gb plan for 3$. 
```

Comment:
Best to try this out (GitLab) before adding it to the wiki.

Reply:
I will leave this on the live version as the risk is minor, but make sure to check that this is a valid piece of advice (unless you already have, in that case disregard these two comments)